lms/djangoapps/courseware/access.py

```diff
@@ -177,15 +197,21 @@ def _can_view_courseware_with_prerequisites(user, course):  # pylint: disable=in
     Arguments:
         user (User): the user whose course access we are checking.
         course (AType): the course for which we are checking access.
-    where AType is CourseDescriptor, CourseOverview, or any other class that
-        represents a course and has the attributes .location and .id.
+            where AType is CourseDescriptor, CourseOverview, or any other class that
+            represents a course and has the attributes .location and .id.
     """
+
+    def _check_if_prerequisites_enabled():
```

Comment:
The name of this method doesn't match its return type.
Rename it to `_is_prerequisites_disabled` and update docstring.